core/txpool/legacypool/legacypool.go

```diff
@@ -171,6 +171,24 @@ var DefaultConfig = Config{
 	ReannounceTime: 10 * 365 * 24 * time.Hour,
 }
 
+// DefaultOpBNBConfig contains the default opBNB configurations for the transaction pool.
+var DefaultOpBNBConfig = Config{
```

Comment:
I think we can keep it same with DefaultConfig

Reply:
Agree, user can set the txpool config manually. Resovled. Please help confirm it again.